modules/ipi-install-preparing-the-bare-metal-node.adoc

```diff
@@ -131,7 +119,6 @@ $ oc -n openshift-machine-api create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/openshift-worker-<num>-network-config-secret created
 secret/openshift-worker-<num>-bmc-secret created
```

Comment:
This is a sample output: 
secret/openshift-worker-2-network created
secret/openshift-worker-2-bmc-secret created
baremetalhost.metal3.io/openshift-worker-2 created